moped-editor/src/views/projects/projectView/ProjectSummary/SubprojectsTable.js

```diff
@@ -52,14 +52,14 @@ const SubprojectsTable = ({ projectId = null, refetchSummaryData }) => {
       width: "15%",
     },
     {
-      title: "Name",
-      field: "project_name",
-      width: "35%",
-      validate: (entry) => !!entry.project_name,
+      title: "Full name",
+      field: "project_name_full",
+      width: "40%",
+      validate: (entry) => (!!entry.project_name),
```

Comment:
I'm not 100% sure this line is the culprit, but I do observe a regression where I can't get the checkbox to turn black when I'm trying to add a new subproject. 
<img width="729" alt="image" src="https://github.com/cityofaustin/atd-moped/assets/3653206/a0fc00e5-6e74-4d4d-a9ad-faa50e309682">

Reply:
yup, you nailed it. i updated the field but didnt update the validate. thanks for catching this!